docs/reference/commandline/service_logs.md

```diff
@@ -17,14 +16,16 @@ advisory: "experimental"
 # service logs
 
 ```Markdown
-Usage:  docker service logs [OPTIONS] SERVICE
+Usage:  docker service logs [OPTIONS] SERVICE|TASK
```

Comment:
Hmmm, so for bash completion I will need a list of all tasks. ATM, this can be done with

    docker service ps ""

Is this supported syntax, will this continue to work?